.travis.yml

```diff
@@ -14,83 +14,27 @@ install:
-    - TOXENV=py27-nobyte
     - TOXENV=py27-xdist
-    - TOXENV=py27-pluggymaster
-    # Specialized factors for py37.
-    - TOXENV=py37-pexpect,py37-trial,py37-numpy
-    - TOXENV=py37-pluggymaster
-    - TOXENV=py37-freeze PYTEST_NO_COVERAGE=1
+  global:
+    CC_TEST_REPORTER_ID=17da631beaabf819740e82ef7edee551fe9a4e41409ffdd81284ad6544a51782
 
 matrix:
   allow_failures:
     - python: '3.8-dev'
       env: TOXENV=py38
 
-jobs:
-  include:
-    # Coverage tracking is slow with pypy, skip it.
-    - env: TOXENV=pypy PYTEST_NO_COVERAGE=1
-      python: 'pypy-5.4'
-      dist: trusty
-    - env: TOXENV=py34
-      python: '3.4'
-    - env: TOXENV=py35
-      python: '3.5'
-    - env: TOXENV=py36
-      python: '3.6'
-    - env: TOXENV=py37
-    - &test-macos
-      language: generic
-      os: osx
-      osx_image: xcode9.4
-      sudo: required
-      install:
-        - python -m pip install --pre tox
-      env: TOXENV=py27
-    - <<: *test-macos
-      env: TOXENV=py37
-      before_install:
-        - brew update
-        - brew upgrade python
-        - brew unlink python
-        - brew link python
-
-    - stage: baseline
-      env: TOXENV=py27-pexpect,py27-trial,py27-numpy
-    - env: TOXENV=py37-xdist
-    - env: TOXENV=linting,docs,doctesting
-      python: '3.7'
-
-    - stage: cron_only
-      env: TOXENV=py38
-      python: '3.8-dev'
-
-    - stage: deploy
-      python: '3.6'
-      env: PYTEST_NO_COVERAGE=1
-      install: pip install -U setuptools setuptools_scm
-      script: skip
-      deploy:
-        provider: pypi
-        user: nicoddemus
-        distributions: sdist bdist_wheel
-        skip_upload_docs: true
-        password:
-          secure: xanTgTUu6XDQVqB/0bwJQXoDMnU5tkwZc5koz6mBkkqZhKdNOi2CLoC1XhiSZ+ah24l4V1E0GAqY5kBBcy9d7NVe4WNg4tD095LsHw+CRU6/HCVIFfyk2IZ+FPAlguesCcUiJSXOrlBF+Wj68wEvLoK7EoRFbJeiZ/f91Ww1sbtDlqXABWGHrmhPJL5Wva7o7+wG7JwJowqdZg1pbQExsCc7b53w4v2RBu3D6TJaTAzHiVsW+nUSI67vKI/uf+cR/OixsTfy37wlHgSwihYmrYLFls3V0bSpahCim3bCgMaFZx8S8xrdgJ++PzBCof2HeflFKvW+VCkoYzGEG4NrTWJoNz6ni4red9GdvfjGH3YCjAKS56h9x58zp2E5rpsb/kVq5/45xzV+dq6JRuhQ1nJWjBC6fSKAc/bfwnuFK3EBxNLkvBssLHvsNjj5XG++cB8DdS9wVGUqjpoK4puaXUWFqy4q3S9F86HEsKNgExtieA9qNx+pCIZVs6JCXZNjr0I5eVNzqJIyggNgJG6RyravsU35t9Zd9doL5g4Y7UKmAGTn1Sz24HQ4sMQgXdm2SyD8gEK5je4tlhUvfGtDvMSlstq71kIn9nRpFnqB6MFlbYSEAZmo8dGbCquoUc++6Rum208wcVbrzzVtGlXB/Ow9AbFMYeAGA0+N/K1e59c=
-        on:
-          tags: true
-          repo: pytest-dev/pytest
-
 before_script:
   - |
     if [[ "$PYTEST_NO_COVERAGE" != 1 ]]; then
       export COVERAGE_FILE="$PWD/.coverage"
       export COVERAGE_PROCESS_START="$PWD/.coveragerc"
       export _PYTEST_TOX_COVERAGE_RUN="coverage run -m"
       export _PYTEST_TOX_EXTRA_DEP=coverage-enable-subprocess
+
+      # Codeclimate.
+      curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+      chmod +x ./cc-test-reporter
+      ./cc-test-reporter before-build -d
```

Comment:
Might need to get removed - not in the example at https://docs.codeclimate.com/docs/travis-ci-test-coverage (the 2nd one).